draft-ietf-ohai-ohttp.md

```diff
@@ -103,7 +103,7 @@ connections to make multiple requests improves performance, but provides
 servers with additional certainty about the identity of clients in a similar
 fashion.
 
-Use of an HTTP proxy can provide a degree of protection against servers
+Use of an HTTP proxy or relay can provide a degree of protection against servers
```

Comment:
```suggestion
Use of an HTTP proxy can provide a degree of protection against servers
```

Reply:
We need something like this to make it clear that relay is basically a proxy, but I'll accept this suggestion and make a further change below.